types/react-transition-group/Transition.d.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, HTMLProps } from "react";
+import { TransitionActions } from "react-transition-group";
+
+export type EndHandler = (node: HTMLElement, done: () => void) => void;
+export type EnterHandler = (node: HTMLElement, isAppearing: boolean) => void;
+export type ExitHandler = (node: HTMLElement) => void;
+
+export interface TransitionProps extends TransitionActions, HTMLProps<any> {
```

Comment:
Hm, true, making it required would make it hard to use with `React.createElement` (JSX would still validate correctly) 😕